common/scripts/configs/master_integration_fields.sql

```diff
@@ -1015,6 +1015,22 @@ do $$
       values (260, '59e71aa80552d20500e76ba9', 'authorization', 'string', false, false, '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-11-03', '2017-11-03');
     end if;
 
+    -- masterIntegrationFields for dclKey
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 262) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (262, '59faf7b67fb80d78e55245bb', 'username', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-31', '2017-10-31');
+    end if;
+
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 263) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (263, '59faf7b67fb80d78e55245bb', 'token', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-31', '2017-10-31');
```

Comment:
This is not secure in the previous integration as well right?

Reply:
no it is not secure in the old one as well